//
// Created by tom on 30/10/16.
//

#ifndef ROBOTICAN_HARDWARE_INTERFACE_DYNAMIXEL_CONTROLLER_H
#define ROBOTICAN_HARDWARE_INTERFACE_DYNAMIXEL_CONTROLLER_H

#include <ros/ros.h>
#include <robotican_hardware_interface/dynamixel_driver.h>

namespace dynamixel_controller {
    class DynamixelController {
    private:

    };
}

#endif //ROBOTICAN_HARDWARE_INTERFACE_DYNAMIXEL_CONTROLLER_H
